### PR TITLE
Added Debian-specific directory

### PR DESCRIPTION
**Pull request by tbazant**
... and removed the old one that seemed ancient to me

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -1,67 +1,102 @@
...
-               libxml2-utils,
-               make,
-               poppler-utils,
-               python-libxml2,
-               python-lxml,
-               xml-core,
-               xmlstarlet,
-               xsltproc
-Standards-Version: 3.9.6
+Maintainer: Filippo Rusconi <lopippo@debian.org>
```
Comment (tomschr) [inline debian/control]: Shouldn't you be the maintainer?

File: debian/copyright [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
+Files-Excluded:
+ debian
+ build
+ .pc
+ autom4te.cache
 
-Files:     *
-Copyright: 2012-2015 SUSE Linux GmbH
+Files: *
+Copyright: 2012-2019 SUSE Linux GmbH
```
Comment (tomschr) [inline debian/copyright]: I think, the current year should be added, right?

```suggestion
Copyright: 2012-2023 SUSE Linux GmbH
```

**Review by tomschr — COMMENTED** [review]
Thanks Tomas! :+1: That was certainly a huge task.

I'm not sure if I can provide good feedback here as I'm not really familiar with Debian. Unfortunately, I cannot test it. If it works for you, that's great!

Maybe as a next step, we could add it to OBS.

Great work! Many thanks! :+1:

File: debian/control (hunk 1 above)
Comment (tbazant) [inline debian/control]:
```suggestion
Maintainer: Tomáš Bažant <tbazant@suse.cz>
```

File: debian/control [hunk 3]
```diff
@@ -1,67 +1,102 @@
...
+ xml-core,
+ xmlstarlet,
+ xsltproc,
+ docbook-to-man
+Standards-Version: 4.6.1
 Homepage: http://opensuse.github.io/daps
-Vcs-Git: https://github.com/openSUSE/daps.git
-Vcs-Browser: https://github.com/openSUSE/daps
+Vcs-Browser: https://salsa.debian.org/lopippo/daps
+Vcs-Git: https://salsa.debian.org/lopippo/daps.git
```
Comment (tbazant) [inline debian/control]:
```suggestion
```

**Review by tbazant — COMMENTED** [review]
Few suggestions accepted

**Pull request closed without merging**